build_runner/pubspec.yaml

```
@@ -1,5 +1,5 @@
 name: build_runner
-version: 1.12.1
+version: 1.12.2-dev
```

lets go ahead and get this out the door, I know a few people are waiting on it
```suggestion
version: 1.12.2
```